templates/default/service-ndvm-hvm

```diff
@@ -23,7 +23,8 @@
     "1": "myself -> 0:5555",
     "2": "0:15555 -> myself:5555",
     "3": "myself -> 0:4878",
-    "4": "my-stubdom -> 0:5100"
+    "4": "my-stubdom -> 0:5100",
+    "5": "myself -> 0:5556"
```

Comment:
Would it be better to change the qtdbd tools to just use the restricted port 5555?

Reply:
Yea we can do that, I'll make the change